src/frontend-build.sh

```diff
@@ -160,6 +160,7 @@ build() {
     -e BRANCH_NAME \
     -e NPM_BUILD_SCRIPT \
     -e YARN_BUILD_SCRIPT \
+    -e RUN_VERIFY \
```

Comment:
please, can we make this a opt-in to skip, instead of disabling it by default?

so instead could be "SKIP_VERIFY"